docs/en/integrations/elastic-package.asciidoc

```diff
@@ -0,0 +1,247 @@
+
+Built packages are stored in the "build/" folder located at the root folder of the local Git repository checkout that contains your package folder. The command will also render the README file in your package folder if there is a corresponding template file present in "_dev/build/docs/README.md". All "_dev" directories under your package will be omitted.
+
+Built packages are served up by the Elastic Package Registry running locally (see "elastic-package stack"). If you want a local package to be served up by the local Elastic Package Registry, make sure to build that package first using "elastic-package build".
+
+Built packages can also be published to the global package registry service.
+
+[discrete]
+=== `elastic-package check`
+
+_Context: package_
+
+Use this command to verify if the package is correct in terms of formatting, validation and building.
+
+It will execute the format, lint, and build commands all at once, in that order.
+
+[discrete]
+=== `elastic-package clean`
+
+_Context: package_
+
+Use this command to clean resources used for building the package.
+
+The command will remove built package files (in build/), files needed for managing the development stack (in ~/.elastic-package/stack/development) and stack service logs (in ~/.elastic-package/tmp/service_logs).
+
+[discrete]
+=== `elastic-package create`
+
+_Context: global_
+
+Use this command to create a new package or add more data streams.
+
+The command can help bootstrap the first draft of a package using embedded package template. It can be used to extend the package with more data streams.
+
+For details on how to create a new package, review the [HOWTO guide](https://github.com/elastic/elastic-package/blob/master/docs/howto/create_new_package.md).
+
+[discrete]
+=== `elastic-package export`
+
+_Context: package_
+
+Use this command to export assets relevant for the package, e.g. Kibana dashboards.
+
+[discrete]
+=== `elastic-package format`
+
+_Context: package_
+
+Use this command to format the package files.
+
+The formatter supports JSON and YAML format, and skips "ingest_pipeline" directories as it's hard to correctly format Handlebars template files. Formatted files are being overwritten.
+
+[discrete]
+=== `elastic-package install`
+
+_Context: package_
+
+Use this command to install the package in Kibana.
+
+The command uses Kibana API to install the package in Kibana. The package must be exposed via the Package Registry.
+
+[discrete]
+=== `elastic-package lint`
+
+_Context: package_
+
+Use this command to validate the contents of a package using the package specification (see: https://github.com/elastic/package-spec).
+
+The command ensures that the package is aligned with the package spec and the README file is up-to-date with its template (if present).
+
+[discrete]
+=== `elastic-package profiles`
+
+_Context: global_
+
+Use this command to add, remove, and manage multiple config profiles.
```

Comment:
It's also a good candidate for an example. Keep in mind this is a feature for advanced users who want to modify the default setup.

Reply:
I'd probably need help on an example here. I'm not sure I understand what the command is used for.